ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -49,10 +49,71 @@
             name: libvirt_manager
             tasks_from: deploy_edpm_compute.yml
 
+        - name: Disable discover_hosts when deploying HCI phase1
+          when: cifmw_edpm_deploy_hci | default(false) | bool
+          ansible.builtin.set_fact:
+            cifmw_edpm_deploy_skip_nova_discover_hosts: true
+
         - name: Deploy EDPM
           ansible.builtin.import_role:
             name: edpm_deploy
 
+- name: Clear edpm hosts facts to force refreshing them
+  hosts: edpm
+  tasks:
+    - name: Clear edpm hosts facts
+      when: cifmw_edpm_deploy_hci | default(false) | bool
+      ansible.builtin.meta: clear_facts
+
+- name: Deploy Ceph
+  vars:
+    storage_network_range: 172.18.0.0/24
+    storage_mgmt_network_range: 172.20.0.0/24
+  ansible.builtin.import_playbook: ceph.yml
+  when: cifmw_edpm_deploy_hci | default(false) | bool
+
+- name: Deploy remaining services when HCI environment
```

Comment:
Planning to move all this logic to a new role, like edpm_deploy_baremetal, since there is more to add here for nova-custom-config before rerunning edpm_deploy phase 2